core/src/main/scala/org/scalawag/jibe/backend/ubuntu/UpdateAptGet.sh

```diff
@@ -0,0 +1,15 @@
+PATH=/bin:/usr/sbin:/usr/bin
+
+lastUpdatedTime=$(stat -c %Y /var/cache/apt/pkgcache.bin)
+currentTime="$(date +'%s')"
+currentCacheAge=$(( currentTime - lastUpdatedTime ))
+maxUpdateAge="$(( $ageInHours * 60 * 60 ))"
+
+echo "Current cache age is $currentCacheAge"
+echo "Maximum acceptable cache age (in seconds) is $maxUpdateAge"
+
+if [ "${currentCacheAge}" -gt "$maxUpdateAge" ]; then
+    echo "Running apt-get update"
+    apt-get update
```

Comment:
I think this will print stuff to stdout, right?  You probably don't need the echo above.